android/KMEA/app/src/main/java/com/keyman/engine/KMKeyboardJSHandler.java

```diff
@@ -149,6 +151,9 @@ public void run() {
 
         // Perform left-deletions
         if (deleteLeft > 0) {
+          if (k.keyboardType == KeyboardType.KEYBOARD_TYPE_INAPP) {
+            KMManager.InAppKeyboardShouldIgnoreSelectionChange = true;
+          }
```

Comment:
Why is this only for inapp?

Reply:
insertText got consolidated to use the SystemKeyboard handler.
The stable-16.0 had this for left-deletions

https://github.com/keymanapp/keyman/blob/869b7900944cbe85ac1d11fa14e5cc3b29556b7d/android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java#L2764-L2773

I suppose I should also set `InAppKeyboardShouldIgnoreTextChange = true;`